SPECS/dnf5/dnf5.spec

```diff
@@ -1,55 +1,101 @@
-%global librepo_version 1.15.0
-%global libsolv_version 0.7.21
-%global sqlite_version 3.35.0
-%global swig_version 4
-%global zchunk_version 0.9.11
+%global project_version_minor 1
+%global project_version_patch 11
+
+Name:           dnf5
+Version:        %{project_version_major}.%{project_version_minor}.%{project_version_patch}
+Release:        1%{?dist}
+Summary:        Command-line package manager
+License:        GPL-2.0-or-later
+URL:            https://github.com/rpm-software-management/dnf5
+Source0:        %{url}/archive/%{version}/dnf5-%{version}.tar.gz
+
+Requires:       libdnf5%{?_isa} = %{version}-%{release}
+Requires:       libdnf5-cli%{?_isa} = %{version}-%{release}
+Recommends:     bash-completion
+
+Provides:       dnf5-command(install)
+Provides:       dnf5-command(upgrade)
+Provides:       dnf5-command(remove)
+Provides:       dnf5-command(distro-sync)
+Provides:       dnf5-command(downgrade)
+Provides:       dnf5-command(reinstall)
+Provides:       dnf5-command(swap)
+Provides:       dnf5-command(mark)
+Provides:       dnf5-command(autoremove)
+Provides:       dnf5-command(check)
+Provides:       dnf5-command(check-upgrade)
+Provides:       dnf5-command(provides)
+
+Provides:       dnf5-command(leaves)
+Provides:       dnf5-command(repoquery)
+Provides:       dnf5-command(search)
+Provides:       dnf5-command(list)
+Provides:       dnf5-command(info)
+
+Provides:       dnf5-command(group)
+Provides:       dnf5-command(environment)
+Provides:       dnf5-command(module)
+Provides:       dnf5-command(history)
+Provides:       dnf5-command(repo)
+Provides:       dnf5-command(advisory)
+
+Provides:       dnf5-command(clean)
+Provides:       dnf5-command(download)
+Provides:       dnf5-command(makecache)
+
+
 # ========== build options ==========
-%bcond_with dnf5daemon_client
-%bcond_with dnf5daemon_server
+
+%bcond_with    dnf5daemon_client
+%bcond_with    dnf5daemon_server
 %bcond_without libdnf_cli
 %bcond_without dnf5
 %bcond_without dnf5_plugins
 %bcond_without plugin_actions
+%bcond_with    plugin_rhsm
 %bcond_without python_plugins_loader
+
 %bcond_without comps
 %bcond_without modulemd
 %bcond_without zchunk
+
 %bcond_with    html
 %bcond_with    man
+
+
 # TODO Go bindings fail to build, disable for now
 %bcond_with    go
 %bcond_with    perl5
 %bcond_without python3
 %bcond_with    ruby
+
 %bcond_with    clang
 %bcond_with    sanitizers
 %bcond_without tests
 %bcond_with    performance_tests
 %bcond_with    dnf5daemon_tests
+
 %if %{with clang}
     %global toolchain clang
 %endif
-Summary:        Command-line package manager
-Name:           dnf5
-Version:        %{project_version_major}.%{project_version_minor}.%{project_version_patch}
-Release:        2%{?dist}
-License:        GPL-2.0-or-later
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
-URL:            https://github.com/rpm-software-management/dnf5
-Source0:        %{url}/archive/%{version}/dnf5-%{version}.tar.gz
```

Comment:
Nit-pick: can we keep this ordering of tags? The spec linter will re-order them for you, if you don't want to do it manually. Not a blocker.

Reply:
Ran the spec linter. Additionally changed `Distribution` to "Azure Linux" instead of Mariner